src/executables/master_server_monitor.hpp

```diff
@@ -0,0 +1,114 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * master_server_monitor.hpp -
+ */
+
+
+#include <cstring>
+#include <thread>
+#include <algorithm>
+#include <memory>
+#include <time.h>
+#include "heartbeat.h"
+#include "system_parameter.h"
+#include "connection_defs.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+class server_monitor
+{
+  public:
+
+    class server_entry;
+    class server_monitor_list;
```

Comment:
`class server_entry;` 불필요한 전방 선언이며, using server_entry_uptr_t를 server_entry class 정의 뒤 쪽으로 옮기면, `class server_monitor_list;` 역시 제거 가능

```suggestion
    class server_entry;
    class server_monitor_list;
```

Reply:
수정하였습니다.